zmsentities/src/Zmsentities/Useraccountprofile.php

```
@@ -0,0 +1,18 @@
+<?php
+
+namespace BO\Zmsentities;
+
+class Useraccountprofile extends Schema\Entity
+{
+    public const PRIMARY = 'username';
```

PHPStan (2.2.7)

[error] Constant BO\Zmsentities\Useraccountprofile::PRIMARY overriding constant BO\Zmsentities\Schema\Entity::PRIMARY (string) should also have native type string.


(classConstant.missingNativeType)